intelmq/bots/experts/time_filter/expert.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+from dateutil import parser
+from intelmq.lib.utils import get_timedelta
+from intelmq.lib.bot import Bot
+from datetime import timezone
+
+
+class TimeFilterExpertBot(Bot):
+    search_field: str = 'time.source'
+    search_from: str  = '1d'
+
+    not_after = None
```

Comment:
This is not a user configurable variable, so please make it private (prepend `__` like `__not_after`). But IIRC a timedelta can also contain a negative value, so we could make call this attribute simply `__delta`, that way we can filter for events in the future and the past.